### PR TITLE
Fix borg lights not turning off when flashed or empd

### PR DESCRIPTION
**Pull request by thegrb93**
## About The Pull Request
Fixes borg light not turning off when flashed because the on_apply was updating the icons before the status was added.

image
## Why It's Good For The Game
Lets people know the borg is stunned
## Changelog
:cl:
fix: Borg lights not turning off when flashed or empd
/:cl:

### CODE REVIEW

File: code/datums/status_effects/debuffs/debuffs.dm [hunk 1]
```diff
@@ -20,10 +20,9 @@
 		duration = set_duration
 	return ..()
 
-/datum/status_effect/incapacitating/on_apply()
+/datum/status_effect/incapacitating/post_apply()
 	if(needs_update_stat || issilicon(owner))
 		owner.update_stat()
```
[the remark this answers is not in the document]
Comment (thegrb93) [reply code/datums/status_effects/debuffs/debuffs.dm:25]: The status effect isn't added until after on_apply (shown in the screenshot) so the update_icons doesn't know the status exists yet. This suggestion doesn't work unfortunately

File: code/datums/status_effects/debuffs/debuffs.dm [hunk 2]
```diff
@@ -20,10 +20,9 @@
 		duration = set_duration
 	return ..()
 
-/datum/status_effect/incapacitating/on_apply()
+/datum/status_effect/incapacitating/post_apply()
```
Comment (MrMelbert) [inline code/datums/status_effects/debuffs/debuffs.dm:23]: I agree with the other review this should just override `on_creation`

File: code/datums/status_effects/debuffs/debuffs.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (thegrb93) [reply code/datums/status_effects/debuffs/debuffs.dm:25]: Alright, got that fixed

**Comment by thegrb93** [comment]
This is working fine from my testing

**Pull request merged**